config/month.map.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const map = {
```

Comment:
maybe `lib` is a better folder?

there is nothing to configure for a user of the library...

its a singleton that will hardly change, and is directly loaded internally